node/parallel-dev/src/chain_spec.rs

```diff
@@ -53,10 +53,6 @@ pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
 
 pub fn development_config() -> Result<ChainSpec, String> {
     let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
-    let mut properties = Map::new();
-    // properties.insert("tokenSymbol".into(), "PARA".into());
-    properties.insert("tokenDecimals".into(), 18.into());
```

Comment:
Cloud you talk more about this?

Reply:
I feel this is not working yet.  because yesterday when I check  polkadotjs settings it's still 12

Reply:
also heiko & parallel didn't configure this yet. 

is it working and how to verify this btw?  because the settings still say decimals 12

Reply:
Because it only is set in parallel-dev. We can have more discussion about this, and I think decimals 12 is Ok.

Reply:
If you modify the `tokenDecimals`, the TOKEN_DECIMAL should as same as with the decimals in the spec settings.